contracts/RiskManagerV1.sol

```diff
@@ -98,6 +98,12 @@ contract RiskManagerV1 is IRiskManager, Auctioneer, Ownable {
     ISignerBondsSwapStrategy public newSignerBondsSwapStrategy;
     uint256 public signerBondsSwapStrategyInitiated;
 
+    // Amount of underwriter tokens granted to the notifier as reward for
+    // `notifyLiquidation` and `notifyLiquidated` calls.
+    uint256 public notifierReward;
+    uint256 public newNotifierReward;
+    uint256 public notifierRewardChangeInitiated;
```

Comment:
I would consider having two rewards: one for `notifyLiquidating` and another for `notifyLiquidated` in case it is needed.

Reply:
I would also consider kicking the can on whether we go for % or with a fixed amount in the following way:
- if the fixed COV amount is specified, we go with the fixed amount,
- otherwise, we go with % of the COV supply.

It should be possible to set the reward to be << 1 %.

I would also consider putting all this logic in a library, similarly how tBTC `Deposit` is split into libraries. This is too much for one contract. And I am thinking about extracting some more of the code from this contract into a library.

Reply:
Just to summarize, we want to have the following:

- `liquidationNotifierRewardAmount`
- `liquidationNotifierRewardPercentage`
- `liquidatedNotifierRewardAmount`
- `liquidatedNotifierRewardPercentage`

All parameters should be governable. `*Amount` parameters should be settable to `0`. `*Percentage` params should be settable to value less than `1%` (is 0.01% at min enough?). If `*Amount` param is `0` we use `*Percentage` of the total COV supply. All this stuff should live in a separate lib. Am I right?

Reply:
Yes. 🧔 

And to clarify:

> (is 0.01% at min enough?)

Let's use 1e18 precision for %, just like we do for cov pool on auction.

Reply:
Ok extracted the library. Please see https://github.com/keep-network/coverage-pools/pull/111/commits/b3d880775271569a41219707797f574e7ad95486. There are still some governance methods missing in the risk manager but it shows the general idea. It also doesn't touch the tests yet. More to come.